examples/Compadre_ReactionDiffusion.cpp

```diff
@@ -23,6 +23,13 @@
 #include <Compadre_ReactionDiffusion_BoundaryConditions.hpp>
 #include <Compadre_ReactionDiffusion_Operator.hpp>
 
+// ####################################
```

Comment:
Drop

Reply:
Add an assertion that it doesn't work for constant basis functions due to calculation of cubature_degree.